src/Options/Applicative/Types.hs

```diff
@@ -150,11 +151,11 @@ data OptVisibility
 -- | Specification for an individual parser option.
 data OptProperties = OptProperties
   { propVisibility :: OptVisibility       -- ^ whether this flag is shown in the brief description
-  , propHelp :: Chunk Doc                 -- ^ help text for this option
+  , propHelp :: Chunk HelpDoc                 -- ^ help text for this option
   , propMetaVar :: String                 -- ^ metavariable for this option
   , propShowDefault :: Maybe String       -- ^ what to show in the help text as the default
   , propShowGlobal :: Bool                -- ^ whether the option is presented in global options text
-  , propDescMod :: Maybe ( Doc -> Doc )   -- ^ a function to run over the brief description
+  , propDescMod :: Maybe ( HelpDoc -> HelpDoc )   -- ^ a function to run over the brief description
```

Comment:
I am not quite sure what these are, therefore also not quite sure if these should be HelpDoc or AnsiDoc.